Structure_Engine/Query/MaterialComposition.cs

```diff
@@ -80,13 +80,15 @@ public static MaterialComposition MaterialComposition(this IAreaElement areaElem
 
         /***************************************************/
 
+        [Description("Returns a SectionProperty's homogeneous MaterialComposition.")]
         public static MaterialComposition MaterialComposition(this ISectionProperty sectionProperty)
         {
             return sectionProperty.IsNull() ? null : (MaterialComposition)Physical.Create.Material(sectionProperty.Material);
         }
 
         /***************************************************/
 
+        [Description("Returns an ConcreteSection's homogeneous MaterialComposition.")]
```

Comment:
This is kind of nit-picking, but the concrete sections material composition is not necessarily homogenous. If it contains reinforcement, it will try to grab it, and if we add a description, think it should cover that.

Reply:
Amended.